framework/LoopMeUnitedSDK/Core/Internal/Controllers/LoopMeVPAIDAdDisplayController.m

```diff
@@ -806,9 +773,7 @@ - (void)videoClientDidBecomeActive:(LoopMeVPAIDVideoClient *)client {
 }
 
 - (void)currentTime:(NSTimeInterval)currentTime percent:(double)percent {
-    if ([[LoopMeViewabilityManager sharedInstance] isViewable:self.delegate.containerView]) {
-        self.viewableTime += currentTime - self.previousVideoTime;
-    }
+    self.viewableTime += currentTime - self.previousVideoTime;
```

Comment:
Does this break the previous logic?

Reply:
No, it just assumes the ad is always viewable.